src/languages/routeros.js

```diff
@@ -0,0 +1,170 @@
+/*
+Language: Microtik RouterOS script
+Author: Ivan Dementev <ivan_div@mail.ru>
+Category: common
```

Comment:
`common` is reserved due to its special meaning, it's what we use for the default CDN build. This language doesn't fall into any categories we have, so no need to specify anything, just drop the line :-)